deploy/ControllerInit.sol

```diff
@@ -150,6 +223,15 @@ library ForeignControllerInit {
 
         // Step 2: Configure all rate limits for controller
 
+        // Sanity check all rate limit data
+        _checkUnlimitedData(data.usdcDepositData,  "usdcDepositData");
+        _checkUnlimitedData(data.usdsDepositData,  "usdsDepositData");
+        _checkUnlimitedData(data.susdsDepositData, "susdsDepositData");
```

Comment:
These should actually not be included as we don't plan to support USDS and sUSDS off the start.

Reply:
Removed these setters